content/en/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -314,6 +314,80 @@ Below are the procedures to set up autocompletion for Bash, Fish, and Zsh.
    rm kubectl-convert kubectl-convert.sha256
    ```
 
+### Uninstall kubectl using package manager
+{{< tabs name="other_kubectl_uninstall" >}}
+{{% tab name="apt-get" %}}
```

Comment:
Another option worth considering is to include instructions for uninstallation using the 'yum' package manager, given that we already have installation instructions for it.